messages/format.go

```diff
@@ -0,0 +1,48 @@
+package messages
+
+import "fmt"
+
+// FormatAmount formats a cent amount as dollars.
+func FormatAmount(v int64) string {
+	f := "-$%d.%02d"
+	if v < 0 {
+		v = -v
+		f = "-" + f
+	}
+
+	return fmt.Sprintf(f, v/100, v%100)
+}
+
+// FormatID returns a compact rendering of ID for use in log messages and other
+// human-readable strings.
+// It returns '<unidentified>' if id is an empty string.
+func FormatID(id string) string {
+	if looksLikeUUID(id) {
+		return id[:uuidSep1] + id[uuidLen:]
+	}
+
+	if id == "" {
+		return "<unidentified>"
+	}
+
+	return id
+}
+
+func looksLikeUUID(id string) bool {
+	if len(id) < uuidLen {
```

Comment:
I am guessing this was probably my suggestion from last time you had a crack at this change? Looking at it now I wonder if we should just use Google's UUID package to parse this since we'll be using it to generate UUIDs when we finally get back to the CLI and/or web interfaces?

Reply:
Yeah I found that in my old first attempt branch from ages ago. Using the Google UUID package to parse this sounds good. I'm not too familiar with Go libraries yet.

Reply:
Yep, all good.  [`uuid.Parse()`](https://godoc.org/github.com/google/uuid#Parse) is what you'd need, unless there's some `IsUUID()` type function I haven't seen.

Reply:
As discussed on Slack, I will remove the compact ID formatter and just show the full ID's as is to keep things simple.